spec.html

```diff
@@ -36802,7 +37383,13 @@ <h1>
           <dl class="header">
           </dl>
           <emu-alg>
-            1. Let _S_ be a String in the form of a |Pattern[~UnicodeMode]| (|Pattern[+UnicodeMode]| if _F_ contains *"u"*) equivalent to _P_ interpreted as UTF-16 encoded Unicode code points (<emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>), in which certain code points are escaped as described below. _S_ may or may not differ from _P_; however, the Abstract Closure that would result from evaluating _S_ as a |Pattern[~UnicodeMode]| (|Pattern[+UnicodeMode]| if _F_ contains *"u"*) must behave identically to the Abstract Closure given by the constructed object's [[RegExpMatcher]] internal slot. Multiple calls to this abstract operation using the same values for _P_ and _F_ must produce identical results.
+            1. If _F_ contains *"v"*, then
+              1. Let _patternSymbol_ be |Pattern[+UnicodeMode, +UnicodeSetsMode]|.
```

Comment:
This is kind of weird. Is there precedent for aliasing grammar nonterminals elsewhere?

Reply:
Yup:
- `Contains` parameter `_symbol_`
- `ComputedPropertyContains` parameter `_symbol_`
- `CreateDynamicFunction` aliases `_exprSym_`, `_bodySym_`, and `_parameterSym_`